Packs/AppendIfNotEmpty/Scripts/AppendIfNotEmpty/AppendIfNotEmpty.yml

```diff
@@ -0,0 +1,36 @@
+args:
+- description: The value to be appended.
+  isArray: true
+  name: value
+- description: Item(s) to append. Comma separated supported.
+  isArray: true
+  name: item
+- auto: PREDEFINED
+  defaultValue: "true"
+  description: Set to true if you want to append item(s) without separating them by
+    comma
```

Comment:
If I understand what you meant, when 'raw' is true and we get a string list (comma separated), the code converts it to a list. Isn't it suppose to be:
`description: Set to *false* if you want to append item(s) without separating them by comma`?

Reply:
The `raw` behaves:
 - true: to add items without separating by comma.
     e.g. value = ["aaa","bbb"], item = "xxx,yyy"  -> return = ["aaa", "bbb", "xxx,yyy"]
     e.g. value = ["aaa","bbb"], item = ["xxx,yyy", "zzz"]  -> return = ["aaa", "bbb", "xxx,yyy", "zzz"]

- false: to add items after separating them into a list by comma.
     e.g. value = ["aaa","bbb"], item = "xxx,yyy"  -> return = ["aaa", "bbb", "xxx", "yyy"]
     e.g. value = ["aaa","bbb"], item = ["xxx,yyy", "zzz"]  -> return = ["aaa", "bbb", "xxx", "yyy", "zzz"]

I hope this makes it clearer for you.

Reply:
Ok got it, Thanks for your example.